com.microsoft.mrtk.core/Utilities/Handedness.cs

```diff
@@ -6,27 +6,30 @@
 namespace Microsoft.MixedReality.Toolkit
 {
     /// <summary>
-    /// Defines in which hand a controller is currently being used.
+    /// Represents handedness; as a bitflag, it can represent left-handedness,
+    /// right-handedness, both, or neither.
     /// </summary>
     [Flags]
     public enum Handedness
     {
         /// <summary>
-        /// No hand specified by the SDK for the controller.
+        /// No handedness. Represents a non-handed controller or object.
         /// </summary>
-        /// <remarks>
-        /// This value is used for controllers such as a gamepad or a mouse.
-        /// </remarks>
         None = 0 << 0,
 
         /// <summary>
-        /// The controller is identified as being provided in a Left hand
+        /// The user's left hand or controller.
         /// </summary>
         Left = 1 << 0,
 
         /// <summary>
-        /// The controller is identified as being provided in a Right hand
+        /// The user's right hand or controller.
         /// </summary>
-        Right = 1 << 1
+        Right = 1 << 1,
+
+        /// <summary>
+        /// Both, or either, hands.
+        /// </summary>
+        Both = Left | Right,
```

Comment:
If we add this, we may want to update some of the `HandednessExtensions`, like https://github.com/microsoft/MixedRealityToolkit-Unity/blob/1405fd95fdf06652207c7c50e2e6e0c4586f5309/com.microsoft.mrtk.core/Utilities/Extensions/HandednessExtensions.cs#L89-L95

some of them assume it will only be Left/Right/None explicitly

Reply:
Hands can only be left or right, there is no NONE hand, not to be confused with controllers or trackers.

Reply:
Revised the `ToXRNode` method to be more explicit, as well as deprecating un-flaggy methods from the extensions